internal/runner/aws_runner.go

```diff
@@ -208,3 +209,12 @@ func (w *AWSFunctionWorkload) handleRunnerTimeout(ctx context.Context,
 		exit <- ExitInfo{255, ErrorRunnerInactivityTimeout}
 	}
 }
+
+// prepareEnvironmentVariables sets the CODEOCEAN variable and unsets some AWS variables.
+func (w *AWSFunctionWorkload) prepareEnvironmentVariables(request *dto.ExecutionRequest) {
+	if request.Environment == nil {
+		request.Environment = make(map[string]string)
+	}
+	request.Environment["CODEOCEAN"] = "true"
+	request.Command = "unset \"${!AWS@}\" && " + request.Command
```

Comment:
That's clever and works perfectly fine in Bash. I just tested that locally with a ZSH, in which it does not work. As we only specify "sh" as a command-line interpreter, are we sure this won't break unnoticed?

Reply:
I have now introduced an e2e test for this. Thanks for your good tip!